samples/core/component_build/component_build.ipynb

```diff
@@ -0,0 +1,368 @@
+    "\n",
+    "In this notebook, we will demo: \n",
+    "\n",
+    "* Defining a KubeFlow pipeline with Python KFP SDK\n",
+    "* Creating an experiment and submitting pipelines to KFP run time enviroment using the KFP SDK \n",
+    "\n",
+    "Reference documentation: \n",
+    "* https://www.kubeflow.org/docs/pipelines/sdk/build-component/\n",
+    "* https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Set your output and project. !!!Must Do before you can proceed!!!\n",
+    "EXPERIMENT_NAME = 'basic_component'\n",
+    "PROJECT_NAME =  'Your-Gcp-Project-Name'                      #'Your-GCP-Project-ID'\n",
+    "OUTPUT_DIR = 'gs://%s-basic-component' % PROJECT_NAME        # A path for asset outputs\n",
+    "BASE_IMAGE = 'google/cloud-sdk:latest'                       # Base image used in various steps of the pipeline\n",
+    "TARGET_IMAGE = 'gcr.io/%s/component:latest' % PROJECT_NAME   # Target image that will include our final code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gsutil mb {OUTPUT_DIR}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create an Experiment in the Pipeline System\n",
+    "\n",
+    "Pipeline system requires an \"Experiment\" to group pipeline runs. You can create a new experiment, or call client.list_experiments() to get existing ones."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Get or create an experiment and submit a pipeline run\n",
+    "import kfp\n",
+    "client = kfp.Client()\n",
+    "\n",
+    "try:\n",
+    "    experiment = client.get_experiment(experiment_name=EXPERIMENT_NAME)\n",
+    "except:\n",
+    "    experiment = client.create_experiment(EXPERIMENT_NAME)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a python function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def add(a: float, b: float) -> float:\n",
+    "    '''Calculates sum of two arguments'''\n",
+    "    \n",
+    "    print(\"Adding two values %s and %s\" %(a, b))\n",
+    "    \n",
+    "    return a + b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build a Component With the Above Function\n",
+    "The return value \"DeployerOp\" represents a step that can be used directly in a pipeline function. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
```

Comment:
You might want to hide those outputs.

Reply:
I generally like to keep the outputs in the notebook so that users have an idea on what to expect.

Reply:
I'm leaning towards keeping the output here to demonstrate what is happening and what is expected if it runs correctly. This can be really useful when the notebook cell refers to something in the output. For example,  after create_run call, we might have documents saying there is a link. Keeping the output link there can be helpful. WDYT?

Reply:
Agreed.